test/bootstrap.php

```diff
@@ -1,5 +0,0 @@
-<?php
-
-\defined('BASE_PATH') || \define('BASE_PATH', \realpath(__DIR__));
-
-require \dirname(__DIR__).'/vendor/autoload.php';
```

Comment:
Can't comment on the rename of the file below, but I wonder why it still works?

Reply:
phpunit can load the `vendor/bootstrap.php` file automatically ;)